templates/default/components/Pre.js

```diff
@@ -20,6 +21,14 @@ export const Pre = ({ children }) => {
     }, 2000);
   };
 
+  if (props.className && props.className === "code-mermaid") {
```

Comment:
We really want some docs here as to why we are doing this and e.g. link to github issue.

I mean i can guess a bit but more info is useful.